source/ocsp-support/ocsp-support.rst

```diff
@@ -76,8 +76,8 @@ invalid, the driver SHOULD end the connection.
     stapled OCSP response, a driver SHOULD end the connection.
 
 2.  If a driver’s TLS library supports Stapled OCSP and the server
-    staples an OCSP response that does not cover the certificate it
-    presents, a driver SHOULD end the connection.
+    staples an OCSP response that is invalid per `RFC 6960 Section 3.2 <https://tools.ietf.org/html/rfc6960#section-3.2>`_,
```

Comment:
Should we keep the language regarding the response not covering the certificate it presents? The RFC notes "The certificate identified in a received response corresponds to the certificate that was identified in the corresponding request;". In the case of a stapled response, there is no explicit request, although I suppose you could argue that implicitly we are requesting the status of the server's cert. I feel like being a bit more explicit here probably wouldn't do any harm though?

Reply:
I originally removed it because of the line you cited from the RFC but I can see how that might cause confusion. I agree that being explicit can't hurt. The new commit reverts that deletion.